init/init.ts

```diff
@@ -0,0 +1,133 @@
+import fs from 'fs'
+import path from 'path'
+import findUp from 'find-up'
+import highlight from 'cli-highlight'
+import inqueier from 'inquirer'
+import { Template } from './Template'
+import ReactScriptsTemplate from './templates/react-scripts'
+import NextTemplate from './templates/next'
+import chalk from 'chalk'
+
+const templates: Record<string, Template> = {
```

Comment:
if everything fails or in general we should give this URL for users to open for anything else https://github.com/bahmutov/cypress-react-unit-test#install